doc/building_applications.rst

```diff
@@ -18,14 +18,23 @@ If ``lld`` does not find the main function for one of the enclaves,
 then an error will be reported.
 
 Building LLVM
----------------
+--------------
 
-Assumed tools: `git`, `cmake`, `ninja`, C compiler
+**Requirements**
+
+LLVM has many software and hardware requirements. They can be found at
+https://llvm.org/docs/GettingStarted.html. In addition to these requirements,
+we require the following packages: ``ccache swig``.
+
+They can be installed on Ubuntu using the following command:
 
 .. code-block:: sh
 
-    # Remeber the top directory
-    $ TOP=$PWD
+    $ sudo apt install ccache swig
```

Comment:
Do you have the command you need to run to install the other software dependencies listed on the GettingStarted page? If so that would be nice to include along with this one.

Reply:
There isn't exactly a recommended command on the page. They list some packages that _should_ already be available, and some that you might need to install. I can add something like 

`sudo apt install cmake gcc python zlib1g make` ?

Reply:
Yeah, that would be enough.